### PR TITLE
April patch translations

### PR DESCRIPTION
**Pull request by gwagwaflamingo**
Translations for ending of chapter The Force Shaking the Rift and beginning of The Aria in the Red Darkness, plus other minor corrections and translations.

ignore whatever is koritemdescription.xml's deal
its fine

its fine

### CODE REVIEW

File: Translated/npcfunctionname.xml [hunk 1]
```diff
@@ -377,10 +377,10 @@
 	<key id="11004843" name="Antiques for Sale" feature="Event" />
 	<key id="11004844" name="이벤트 상인" feature="Event" locale="CN" />
 	<key id="11004845" name="Event Shop" feature="Event" locale="KR" />
-	<key id="11004876" name="로얄 뮤직 아카데미 행정관" feature="EpicApocalypse04" />
+	<key id="11004876" name="Music Academy Head" feature="EpicApocalypse04" />
```
Comment (craftingmod) [inline Translated/npcfunctionname.xml:380]: Royal Music Academy Administrator?

**Review by craftingmod — COMMENTED** [review]
Nice translation~

I'm confusing about `Red Dark` or `Red Darkness` or `Red Dark Matter` or `Red Dark Light` but..
Nobody would care because that's just red dark matters. (No one is reading npc name of that red dark matters btw)

File: Translated/npcfunctionname.xml (hunk 1 above)
Comment (gwagwaflamingo) [reply Translated/npcfunctionname.xml:380]: While I _do_ prefer 'Royal Music Academy Administrator', 'Music Academy Head' is the official translation from GMS2

**Comment by gwagwaflamingo** [comment]
> Nice translation~

Thank you!

> I'm confusing about `Red Dark` or `Red Darkness` or `Red Dark Matter` or `Red Dark Light` but.. Nobody would care because that's just red dark matters. (No one is reading npc name of that red dark matters btw)

Yeah, it's complicated lmao. The wording really changes depending on the context.
'Dark red' refers to the color of the Red Dark Matter; a tone of red that is dark;
'Red darkness' is just another way they refer to the Red Dark Matter; a way that describes what the Red Dark Matter is;
'Red Dark Matter' is a Dark Matter that is red due to the influence of the Red Lapenta;
'Red light' refers to the glow of the Red Dark Matter.

**Pull request merged**